### PR TITLE
Add exams documentation

### PR DESCRIPTION
**Pull request by kashav**
https://github.com/cobalt-uoft/documentation/issues/24

### CODE REVIEW

File: endpoints/exams/show.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# GET exams/:d
```
Comment (qasim) [inline endpoints/exams/show.md]: :id

File: endpoints/exams/show.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+https:/cobalt.qas.im/api/1.0/exams/:id
+```
+
+## Parameters
+
+`key` _(required)_
+Your unique API key. If you do not provide this, you must set the `Authorization` header to your API key instead.
+- - -
```
Comment (qasim) [inline endpoints/exams/show.md:15]: `:id`

File: endpoints/exams/show.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+`:d` _(required)_
+The unique identifier for the exam.
+
+## Example
+
+```
+https:/cobalt.qas.im/api/1.0/exams/CSC148H1F20159DEC15
```
Comment (qasim) [inline endpoints/exams/show.md]: `https://`

File: endpoints/exams/show.md (hunk 2 above)
Comment (qasim) [inline endpoints/exams/show.md:15]: Whoops, on mobile and got the wrong line, should be the one below.

**Comment by kashav** [comment]
@qasim Looks like I deleted a bunch of text after those colons, fixed!

#movefastandbreakthings

**Comment by qasim** [comment]
👌

**Pull request merged**